NDTensors/ext/NDTensorsCUDAExt/adapt.jl

```diff
@@ -11,18 +13,22 @@ function cu(xs; unified::Bool=false)
   )
 end
 
-buffertype(::NDTensorCuArrayAdaptor{B}) where {B} = B
+function TypeParameterAccessors.position(
+  ::Type{<:NDTensorCuArrayAdaptor}, ::typeof(NDTensors.storagemode)
```

Comment:
We should rename this from `NDTensorCuArrayAdaptor` to `CuArrayAdaptor` and move it to `CUDAExtensions`.